src/config.js

```diff
@@ -3,43 +3,29 @@ const TITLE_FILE = '_title.md';
 const SUMMARY_FILE = 'summary.md';
 const IMAGE_EXTENSIONS = ['.png'];
 const MD_EXTENSION = '.md';
-const HTML_EXTENSION = '.html';
 const MARKDOWN_EXTENSIONS = [MD_EXTENSION];
-const HTML_DIRECTORY = 'html';
 const MARKDOWN_DIRECTORY = 'markdown';
 const MARKDOWN_DIRECTORY_FOR_PDF = 'markdown_for_pdf';
 const PDF_DIRECTORY = 'pdf';
 const DIRECTORY = 'directory';
 const PDF_FILE = 'output.pdf';
 const STYLES_DIRECTORY = 'styles';
 const PDF_STYLES_FILE = 'pdf-styles.css';
-const HTML_STYLES_FILE = 'html-styles.css';
 // Markdown supports a maximum of 6 nesting levels
 const MAX_SUPPORTED_SECTION_LEVELS = 6;
 const SECTION_LEVELS = 6;
 const INTRO_FILE = '_intro.md';
 const PAGE_BREAK = '<div class="page-break"></div>';
 const SUMMARY_FILE_TITLE = '# Table of Contents';
-const IGNORE = [TITLE_FILE, MARKDOWN_DIRECTORY, HTML_DIRECTORY, CONTENT_FILE, INTRO_FILE, SUMMARY_FILE];
-const STYLES_HREF_MARKER = '___STYLES_HREF___';
-const HTML_HEADER = `
-<html>
-  <head>
-    <link href="https://fonts.googleapis.com/css?family=Lato:400,900,700,700italic,400italic,300italic,300,100italic,100,900italic" rel="stylesheet" type="text/css">
-    <link href="https://fonts.googleapis.com/css?family=Dosis:400,500,700,800,600,300,200" rel="stylesheet" type="text/css">
-    <link rel="stylesheet" type="text/css" href="${STYLES_HREF_MARKER}">
-  </head>
-  <body>`;
-const HTML_FOOTER = `
-  </body>
-</html>`;
+const IGNORE = [TITLE_FILE, MARKDOWN_DIRECTORY, CONTENT_FILE, INTRO_FILE, SUMMARY_FILE];
+const SINGLE_MARKDOWN_FILE = 'content.md';
```

Comment:
This will be the default name of the content file. @Incei should it be named differently?

Reply:
Would be nicer to name it report.md (assuming this is the complete report merged as 1 file, which can be directly converted to pdf as is).